### DISCUSSION TITLE
Easy game deletes

### DISCUSSION THREAD

**Discussion by alshark** [OnionUI/Onion#586] [Ideas]
I'd love to see faster way to delete games.
Maybe when going through a list of games, press the left arrow to select a games (basically enable multiselect) and then do a delete on all selected somehow. 
I don't know - might be only me, but I would find this extremely helpfull to cleanup my collections of thousends of games (I totally ripped from cartridges myself btw ;) ).

**Comment by Olywa** [comment]
It's a nice idea but not something that can be easily implemented currently as the main interface (and game list screens) are governed by the Miyoo binaries for which they have not shared the source code. 

Today only individual game deletion is possible (by highlighting the game and pressing `SELECT` > `Delete`).

Well done on dumping all those cartridges though! ;)

**Reply by ruben2000de** [reply]
I noticed, that deleting currently only de-listing it. My wish where, that there would be an option in the tweaks let me choose to set if it would delete it from the list only or also would delete the rom completely from the storage.

**Comment by dreambrace** [comment]
An alternative solution to this would be to use the file explorer app, which you can get from the package manager. It's not as pretty (especially if you have all the preview images), but it saves you from fiddling with your sd card.
